test/e2e-common.sh

```diff
@@ -216,9 +216,11 @@ function install_knative_serving_standard() {
   echo ">> Bringing up Serving"
   kubectl apply -f "${INSTALL_RELEASE_YAML}" || return 1
 
-  if ((RECONCILE_GATEWAY)); then
+  if (( RECONCILE_GATEWAY )); then
     echo ">>Turning on reconcileExternalGateway."
-    kubectl get cm config-istio -n knative-serving -o yaml | sed 's/\ \ reconcileExternalGateway: \"false\"/reconcileExternalGateway: \"true\"/g' | kubectl replace -f -
+    kubectl get cm config-istio -n knative-serving -o yaml | \
+      sed 's/\ \ reconcileExternalGateway: \"false\"/reconcileExternalGateway: \"true\"/g' |\
```

Comment:
not sure why you escape some spaces but not all, is it because these are leading spaces?

Reply:
This is because I need to remove the first two leading spaces (https://github.com/knative/serving/blob/master/config/config-istio.yaml#L65) in order to actually make the new value effective.

Reply:
I don't think you need to escape space
```suggestion
      sed 's/  reconcileExternalGateway: \"false\"/reconcileExternalGateway: \"true\"/g' |\
```
would this work?

Reply:
Done.